### PR TITLE
Print debugging information about the repo used.

### PR DESCRIPTION
**Pull request by schmmd**
Although I created a new repository (https://github.com/schmmd/tango-test) when I run the Beaker Executor `allenai/tango` is cloned and my source files are not found.  I think these debugging statements might be helpful, because otherwise it's not clear what repository is being used.

### CODE REVIEW

File: tango/integrations/beaker/executor.py [hunk 1]
```diff
@@ -610,6 +610,7 @@ def _build_experiment_spec(self, step_graph: StepGraph, step_name: str) -> Exper
             )
         try:
             github_account, github_repo = self._parse_git_remote(step_info.environment.git.remote)
+            print(f"Using GitHub repository '{github_account}/{github_repo}'")
```
Comment (schmmd) [inline tango/integrations/beaker/executor.py:613]: Unfortunately this prints once for each step.

**Comment by schmmd** [comment]
The original problem that I had was that I had copied a workspace from one repo into another, so it was using the original github URL rather than the URL of the repository I was currently using, which was quite confusing!

File: tango/integrations/beaker/executor.py (hunk 1 above)
Comment (schmmd) [reply tango/integrations/beaker/executor.py:613]:
```
Starting new run game-horse                                                                                                                                                                                         
Server started at http://localhost:8080/run/game-horse                                                                                                                                                              
Using GitHub repository 'schmmd/tango-test'
Using GitHub repository 'schmmd/tango-test'
Using GitHub repository 'schmmd/tango-test'
Submitted Beaker experiment https://beaker.org/ex/01GA4SGXAHKG1E5674JPPEKP0V for step "sin" (SineStep-31c1K9dVdi3f2FJKsSPnBCF4iLrFoiZ8)                                                                             
Submitted Beaker experiment https://beaker.org/ex/01GA4SGXHX2HKM97NACXMTD9BP for step "cos" (CosineStep-31c1K9dVdi3f2FJKsSPnBCF4iLrFoiZ8)                                                                           
Submitted Beaker experiment https://beaker.org/ex/01GA4SGXS8A9M9SQYFVMKDD4WD for step "i_times_pi" (MultiplyStep-2MdPTBJNfg7ExAxDbs6WsknMKvXh9zBw)   
```

**Review by epwalsh — APPROVED** [review]
LGTM, I just changed it to a logging statement instead of a raw `print()`.

**Pull request merged**